.azure-pipelines/stage/publish.yml

```diff
@@ -226,8 +226,83 @@ jobs:
           set -e
           rm -rf $(Build.StagingDirectory)/.gnupg
 
+- job: docs
+  displayName: Publish docs
+  dependsOn: []
+  condition: |
+    and(not(canceled()),
+        eq(${{ parameters.publishDocs }}, 'true'))
+  pool:
+    vmImage: "ubuntu-20.04"
+  steps:
+  - template: ../bazel.yml
+    parameters:
+      ciTarget: docs
+      cacheVersion: $(cacheKeyBazel)
+      publishEnvoy: false
+      publishTestResults: false
+      stepsPost:
+
+      - script: |
+          ci/run_envoy_docker.sh 'ci/do_ci.sh dockerhub-publish'
+        condition: |
+          and(not(canceled()),
+              eq(${{ parameters.publishDockerhub }}, 'true'))
+        displayName: "Publish Dockerhub description and README"
+        env:
+          ENVOY_DOCKER_BUILD_DIR: $(Build.StagingDirectory)
+          ENVOY_RBE: "1"
+          BAZEL_BUILD_EXTRA_OPTIONS: "--config=remote-ci --jobs=$(RbeJobs)"
+          BAZEL_REMOTE_CACHE: grpcs://remotebuildexecution.googleapis.com
+          BAZEL_REMOTE_INSTANCE: projects/envoy-ci/instances/default_instance
+          GCP_SERVICE_ACCOUNT_KEY: ${{ parameters.authGCP }}
+          GCS_ARTIFACT_BUCKET: ${{ parameters.bucketGCP }}
+          DOCKERHUB_USERNAME: ${{ parameters.authDockerUser }}
+          DOCKERHUB_PASSWORD: ${{ parameters.authDockerPassword }}
+
+      # Trigger Netlify rebuild of latest docs
```

Comment:
nit: I think this coment should be add to `i/run_envoy_docker.sh 'ci/do_ci.sh docs-publish-latest'`

Reply:
its here because the preceeding step (ie uploading to gcs) is done in order to publish to netlify - so i think this is more correct (i moved it here specifically for this reason)